acceptance: skip TestComposeGSSPython

rickystewart opened the pull request:
This is now broken due to a new version of `psql` being pulled in that
is more strict about cert ownership.

See https://github.com/cockroachdb/cockroach/issues/81254.

Release note: None

cockroach-teamcity commented:
<!-- Reviewable:start -->
This change is [<img src="https://reviewable.io/review_button.png" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/cockroachdb/cockroach/81244)
<!-- Reviewable:end -->

kenliu-crl reviewed (approved):
> Reviewed 1 of 1 files at r1, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/cockroachdb/cockroach/81244)* status: :shipit: complete! 0 of 0 LGTMs obtained (waiting on @rickystewart)



<!-- Sent from Reviewable.io -->

rickystewart commented:
bors r=otan,jlinder,kenliu-crl

rickystewart commented:
bors r=otan,jlinder,kenliu-crl p=9999

rickystewart commented:
bors r-

rickystewart commented:
bors r=otan,jlinder,kenliu-crl p=9999

otan commented:
(add a `single on` for good measure and you smash everyone already in the queue :D)

rickystewart committed:
acceptance: skip `TestComposeGSSPython`

This is now broken due to a new version of `psql` being pulled in that
is more strict about cert ownership.

See #81254.

Release note: None

rickystewart commented:
bors r=otan,jlinder,kenliu-crl p=9999 single on

otan commented:
blathers backport 22.1.0
